<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>244</width>
    <height>407</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>721</width>
      <height>571</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="login">
     <attribute name="title">
      <string>Login</string>
     </attribute>
     <widget class="QLineEdit" name="login_input_usuario">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>60</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="login_input_senha">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>130</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="login_btn_entrar">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>190</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Entrar</string>
      </property>
     </widget>
     <widget class="QLabel" name="login_erro">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>180</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="login_label_acesso">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>290</y>
        <width>221</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="login_btn_cadastrar">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>230</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Cadastrar</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="sistema">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <attribute name="title">
      <string>Sistema</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>110</y>
        <width>151</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Login realizado com sucesso</string>
      </property>
     </widget>
     <widget class="QPushButton" name="sistema_btn_bloquear">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>290</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Bloquear</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
